minibroker/scripts/build-minibroker.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+set -e
+set -x
+
+# Prepare minibroker dir for building docker image
+cp -R src minibroker/
+
+# Build the executable
+mkdir -p gopath/src/github.com/osbkit
+cp -R src gopath/src/github.com/osbkit/minibroker
+export GOPATH=$PWD/gopath
+cd gopath/src/github.com/osbkit/minibroker
+make build-linux
```

Comment:
`make -C gopath/src/github.com/osbkit/minibroker build-linux`?

Reply:
Fixed in 8dea0af